backend/django_core/templates/anime/index.html

```diff
@@ -344,202 +334,69 @@
-                                <circle cx="5" cy="5" r="5" fill="currentColor"></circle>
-                            </svg>
-                            <span>19/24 eps</span>
-                        </anime_info>
-                    </div>
-                </anime-info>
-            </a>
-            <a href="/anime/mal/6/episode/11" class="relative" aria-expanded="false">
-                <div class="h-60 w-full rounded-lg object-cover object-center md:h-[20vw] md:rounded-[0.5vw]">
-                    <img class="h-60 w-full rounded-lg object-cover object-center md:h-[20vw] md:rounded-[0.5vw]" src="/images/cover/jjk.webp" alt="Jujutsu Kaisen 2nd season" style="" loading="lazy" />
-                </div>
-                <anime-info class="absolute inset-x-0 bottom-0 rounded-b-lg backdrop-blur md:rounded-b-[0.5vw]">
-                    <div class="flex flex-col gap-1 bg-surface-900/90 p-3 md:gap-[0.35vw] md:p-[1vw]">
-                        <scroll-area class="block h-full w-full overflow-y-scroll overscroll-y-contain scrollbar-thin [scrollbar-color:_rgba(255,255,255,0.12)_transparent]">
-                            <div>
-                                <div class="flex overflow-hidden text-sm font-semibold duration-300 ease-in-out scrollbar-none hover:max-h-[10vw] hover:overflow-y-scroll md:max-h-[1.35vw] md:text-[1vw] md:leading-[1.35vw]">
-                                    <span class="line-clamp-1 md:line-clamp-none">Jujutsu Kaisen 2nd season</span>
-                                </div>
-                            </div>
-                        </scroll-area>
-                        <anime_info class="flex items-center gap-2 text-xs leading-none text-surface-50 md:gap-[0.5vw] md:text-[0.8vw]">
-                            <span class="hidden md:flex">Watching</span>
-                            <svg class="hidden opacity-75 md:flex md:w-[0.25vw]" viewBox="0 0 10 10" fill="none" xmlns="http://www.w3.org/2000/svg">
-                                <circle cx="5" cy="5" r="5" fill="currentColor"></circle>
-                            </svg>
-                            <span>11/12 eps</span>
-                        </anime_info>
-                    </div>
-                </anime-info>
-            </a>
-            <a href="/anime/mal/7/episode/9" class="relative" aria-expanded="false">
-                <div class="h-60 w-full rounded-lg object-cover object-center md:h-[20vw] md:rounded-[0.5vw]">
-                    <img class="h-60 w-full rounded-lg object-cover object-center md:h-[20vw] md:rounded-[0.5vw]" src="/images/ToLOVERu.avif" alt="To LOVE-Ru" style="" loading="lazy" />
+                        </anime-info>
+                    </button>
+                    <ul
+                        tabindex="0"
+                        class="dropdown-content z-[1] top-0 left-[13.85vw] bg-neutral md:p-[0.5vw] rounded-[0.5vw] md:w-[10vw]"
```

Comment:
`z-[1]`

You should never try to do things like this.

Reply:
Nah I dont, Its Daisy's work